backend/runner/runner.go

```diff
@@ -107,6 +117,18 @@ func Start(ctx context.Context, config Config) error {
 	go rpc.RetryStreamingClientStream(ctx, backoff.Backoff{}, controllerClient.RegisterRunner, svc.registrationLoop)
 	go rpc.RetryStreamingClientStream(ctx, backoff.Backoff{}, controllerClient.StreamDeploymentLogs, svc.streamLogsLoop)
 
+	instanceCounter, err = meter.Int64UpDownCounter("ftl.sys.runner.instance",
+		metric.WithDescription("number of runner instances"),
+		metric.WithUnit("{count}"))
+
+	if err != nil {
+		panic(err)
```

Comment:
We probably don't want to crash the whole runner when otel instrumentation fails. Could you take a look at how the existing code users the logger from ctx?

Reply:
noted, this meter is also no longer in scope. will apply this feedback in other metric inits